### ISSUE TITLE
Unwanted Frame added to plot

### ISSUE THREAD

**Issue by cmohamma** [ropensci/plotly#749]
I am attempting to publish a plotly plot to my account. The following code has worked for months:
Say `p` is ggplot2 object.

```
gg <- plotly_build(p)
plotly_POST(gg, filename = "plot1")
```

However, the above code now produces frames around the plot that did not exist before.

![screen shot 2016-10-04 at 4 59 51 pm](https://cloud.githubusercontent.com/assets/5499407/19096853/c2b83934-8a54-11e6-91b1-f5bab00d7c12.png)

**Comment by cmohamma** [comment]
Also, I forgot to mention. I get the following warning after running` plotly_POST(gg, filename = "example.plot")`

```
No encoding supplied: defaulting to UTF-8.
The draw time for this plot will be slow for all clients.FALSE
Success! Modified your plotly here -> https://plot.ly/~cmohamma/38
Warning message:
Estimated Draw Time Too Long 
```

**Comment by cmohamma** [comment]
Out of nowhere, plotly works. See image below. Note, I did not republish the fixed map. When I went to my account it was simply published correctly (almost).

![screen shot 2016-10-04 at 4 58 08 pm](https://cloud.githubusercontent.com/assets/5499407/19096758/dae3b0ca-8a53-11e6-8983-0df30bec4e16.png)

I say almost because now there is a frame around the produced image whereas no such frame exist before. Any ideas how to get rid of it?

![screen shot 2016-10-04 at 4 59 51 pm](https://cloud.githubusercontent.com/assets/5499407/19096771/00d510a8-8a54-11e6-98f9-33a0b0a5afdd.png)
